### ISSUE TITLE
mismatched CocoaPods version Gemfile.lock vs Podfile.lock

### ISSUE THREAD

**Issue by sebbean** [thoughtbot/Tropos#161]
Podfile.lock created with version 0.38.2: https://github.com/thoughtbot/Tropos/blob/master/Podfile.lock#L48
Gemfile.lock has cocoapods versioned at 0.37.2
https://github.com/thoughtbot/Tropos/blob/master/Gemfile.lock#L11

Resulting in this warning in `bin/setup`:

```
[!] The version of CocoaPods used to generate the lockfile (0.38.2) is higher than the version of the current executable (0.37.2). Incompatibility issues may arise.
```

and possibly related error later in `bin/setup` : `Generating Pods project`
<img width="1293" alt="screen shot 2015-11-30 at 1 59 32 pm" src="https://cloud.githubusercontent.com/assets/423997/11463792/acf8ec2a-976a-11e5-9ba7-1031f5fa72c5.png">

And, lastly, in workspace:

<img width="841" alt="screen shot 2015-11-30 at 2 02 54 pm" src="https://cloud.githubusercontent.com/assets/423997/11463819/16d53fae-976b-11e5-8a95-d2d3d706ed69.png">

Seeing if I can whip up a quick fix.

**Comment by sebbean** [comment]
Related-ish: 

Have you guys ever broken down how you use CocoaPods vs Carthage? I've never really seen both being used in one project. 

Have you ever considered going pure submodules? I tried Carthage during the CocoaPods drought a year or so back. Didn't quite work for me at the time.

Found that git submodules actually isn't too bad. It allowed me to really understand how all the code is hooked into the host project, greatly improving the debugging process and removes a whole other set of dependencies (other than semver which could be achieved using tags I'm sure)
